src/v2/view-builder/components/Link.js

```diff
@@ -21,10 +21,14 @@ export default View.extend({
 
   attributes() {
     let href = this.options.href || '#';
+    const rel = (this.options.target && !this.options.rel)
+      ? 'noopener noreferrer'
+      : this.options.rel;
```

Comment:
Focus here

Reply:
So this approach would default `rel` to `"noopener noreferrer"` even when `target` isn't `"_blank"`? I assume there's no drawbacks to that?

Reply:
I'll limit it to `target="_blank"`